cmd/lakectl/cmd/cat_sst.go

```diff
@@ -16,49 +15,73 @@ import (
 	"google.golang.org/protobuf/proto"
 )
 
-func readStdin() (pebblesst.ReadableFile, error) {
-	// test if stdin is seekable
-	if isSeekable(os.Stdin) {
-		return os.Stdin, nil
-	}
-	// not seekable - read it into a temp file
-	fh, err := os.CreateTemp("", "cat-sst-*")
-	if err != nil {
-		return nil, fmt.Errorf("error creating tempfile: %w", err)
-	}
-	_, err = io.Copy(fh, os.Stdin)
-	if err != nil {
-		return nil, fmt.Errorf("error copying from stdin to tempfile: %w", err)
-	}
-	err = os.Remove(fh.Name())
-	if err != nil {
-		return nil, fmt.Errorf("could not unlink temp file %s: %w", fh.Name(), err)
-	}
-	return fh, nil
+const sstCatTemplate = `{{ .Table | table }}`
+
+var catSstCmd = &cobra.Command{
+	Use:    "cat-sst <sst-file>",
+	Short:  "Explore lakeFS .sst files",
+	Hidden: true,
+	Run: func(cmd *cobra.Command, args []string) {
+		amount, _ := cmd.Flags().GetInt("amount")
+		filePath, _ := cmd.Flags().GetString("file")
```

Comment:
```suggestion
		filePath, _ := Must(cmd.Flags().GetString("file"))
```

Reply:
applied it on all ignored err

Reply:
After fixing this one it looks like we found some bug - https://github.com/treeverse/lakeFS/actions/runs/5671553692/job/15369013035?pr=6260